clippy_lints/src/methods/unwrap_used.rs

```diff
@@ -24,17 +24,22 @@ pub(super) fn check(cx: &LateContext<'_>, expr: &hir::Expr<'_>, recv: &hir::Expr
     }
 
     if let Some((lint, kind, none_value)) = mess {
+        let help = if is_lint_allowed(cx, EXPECT_USED, expr.hir_id) {
+            format!(
+                "if you don't want to handle the `{none_value}` case gracefully, consider \
+                using `expect()` to provide a better panic message"
+            )
+        } else {
+            format!("if this value is an `{none_value}`, it will panic")
```

Comment:
On `Option`, it would be better to be "a \`None\`" not "an \`None\`".

Reply:
Nice catch. I also fixed it in `expect_used`